### ISSUE TITLE
Dials no longer open in new tab

### ISSUE THREAD

**Issue by masowards** [fastaddons/GroupSpeedDial#58]
**Describe the bug**
WHen I click on a dial it usually open in a new tab as per my FF setting, but now it does not switch to that tab automatically

**Steps To Reproduce**
open  New tab and select a Group Speed Dial click on it.

**Expected behavior**
The browser should switch to the new tab and load the page from the clicked dial

**Device info:**
 - operating system:  Win10 Pro 64bit  build 10.0.19043 
 - Browser and version: FF 90.0.2 (64-bit)
 - GroupSpeedDial version:  16.0

**Additional info**
Add any other info about the problem here.

**Comment by masowards** [comment]
I just updated to the latest FF and noticed this problem.  I checked my settings and I have 
Open links in tabs instead of new windows: checked  & 
When you open a link, image or media in a new tab, switch to it immediately: checked
When I right click on a link in a page and select open in new tab the browser does switch to the new tab, but not when I click on a dial.  The new tab is opened, but not switched to.
